.github/workflows/kernel-arm64-debug.config

```diff
@@ -766,6 +766,7 @@ CONFIG_RUST_EXAMPLE=y
 CONFIG_RUST_EXAMPLE_2=y
 CONFIG_RUST_EXAMPLE_3=m
 CONFIG_RUST_EXAMPLE_4=m
+CONFIG_RUST_EXAMPLE_5=m
```

Comment:
By adding an example, I meant to the other `rust_example`s, rather than adding a new example. :)

(We have 4 of them because I wanted to run more than 1 builtin + more than 1 module).